meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor

```diff
@@ -61,8 +61,8 @@ if [ -z "$API_ENDPOINT" -o -z "$CONFIG_PATH" ]; then
 fi
 
 # If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
-read DEVICE_API_KEY DEVICE_ID <<<$(jq -r '.apiKey // .deviceApiKey,.deviceId' $CONFIG_PATH)
-if [ -z "$DEVICE_API_KEY" -o -z "$DEVICE_ID" ]; then
+_device_api_key=${PROVISIONING_API_KEY:-DEVICE_API_KEY}
```

Comment:
Shouldn't this just fail if there's no device apikey? After all this should only run after registration.